From 4dee682d91ddb482aa1ec3d5582387bca15ead26 Mon Sep 17 00:00:00 2001
From: Alexandru Costache <alexandru@balena.io>
Date: Tue, 1 Jun 2021 13:14:20 +0200
Subject: [PATCH] Enable i2c4 in uboot

We used to add the pinctrl for i2c4grp
since it wasn't present in the fsl-imx8mm-var-dart.dts
file. However, the dtb name has changed in upstream so
we now ensure the same configuration is being applied
to the new dts.

Upstream-status: Inappropriate[configuration]
Signed-off-by: Alexandru Costache <alexandru@balena.io>
---
 arch/arm/dts/imx8mm-var-dart-customboard.dts | 11 -----------
 1 file changed, 11 deletions(-)

diff --git a/arch/arm/dts/imx8mm-var-dart-customboard.dts b/arch/arm/dts/imx8mm-var-dart-customboard.dts
index c0a6012676..e3377e1d83 100644
--- a/arch/arm/dts/imx8mm-var-dart-customboard.dts
+++ b/arch/arm/dts/imx8mm-var-dart-customboard.dts
@@ -60,14 +60,6 @@
 		>;
 	};
 
-	pinctrl_i2c4_gpio: i2c4grp-gpio {
-		fsl,pins = <
-			MX8MM_IOMUXC_I2C4_SCL_GPIO5_IO20		0x1c3
-			MX8MM_IOMUXC_I2C4_SDA_GPIO5_IO21		0x1c3
-		>;
-	};
-
-
 	pinctrl_reg_usdhc2_vmmc: regulator-usdhc2 {
 		fsl,pins = <
 			MX8MM_IOMUXC_SD2_RESET_B_GPIO2_IO19		0x41
@@ -132,9 +124,6 @@
 	clock-frequency = <100000>;
 	pinctrl-names = "default", "gpio";
 	pinctrl-0 = <&pinctrl_i2c4>;
-	pinctrl-1 = <&pinctrl_i2c4_gpio>;
-	scl-gpios = <&gpio5 20 GPIO_ACTIVE_HIGH>;
-	sda-gpios = <&gpio5 21 GPIO_ACTIVE_HIGH>;
 	status = "okay";
 };
 
-- 
2.17.1

